app/registries/foreman/access_permissions.rb

```diff
@@ -8,6 +8,7 @@
       :notification_recipients => [:index, :update, :destroy, :update_group_as_read, :destroy_group ] }, :public => true
     map.permission :api_status, { :"api/v2/home" => [:status]}, :public => true
     map.permission :about_index, { :about => [:index] }, :public => true
+    map.permission :user_menu, { :"api/v2/users" => [:menu] }, :public => true
```

Comment:
My lack of Rails knowledge, but this path differs from the URL path. Is that because you're routing it to the users controller?

Reply:
It is because `:menu` action in `api/v2/users` controller will be mapped to the `api/v2/menu` path and this says I want to add a 'public' permission for that controller and action.